app/post/controller.js

```diff
@@ -21,3 +21,8 @@ module.exports.show = function (req, res) {
     res.render('post/show', {post: post});
   });
 };
+
+module.exports.parse = function (req, res, next) {
```

Comment:
If you are returning an array of words, I'd change this function name to "parseWords" or something.